lib/uo_cli.rb

```diff
@@ -204,6 +204,10 @@ def opt_opendss
           "Example: uo opendss --scenario baseline_scenario.csv --feature example_project.json --end-time '2017/01/16 01:00:00'\n" \
           "Ensure you have quotes around the timestamp, to allow for the space between date & time.", type: String
 
+          opt :upgrade, "\nUpgrade undersized transformers\n" \
+          "Optional, defaults to false if not provided\n" \
```

Comment:
looks great! can you just add a short name for this? i like to have those explicitly defined in case they randomly change in the future (when we add more flags). thanks!